Tasks/NuGetAuthenticateV0/task.json

```diff
@@ -0,0 +1,48 @@
+{
+    "id": "f5fd8599-ccfa-4d6e-b965-4d14bed7097b",
+    "name": "NuGetAuthenticate",
+    "friendlyName": "NuGet authenticate",
+    "description": "Configure nuget.exe or dotnet to authenticate with Azure Artifacts and other NuGet repositories. Requires NuGet >= 4.8.5385, or dotnet >= 2.1.400",
+    "author": "Microsoft Corporation",
+    "helpUrl": "https://aka.ms/NuGetAuthenticateTask",
+    "helpMarkDown": "[Learn more about this task](https://aka.ms/NuGetAuthenticateTask)",
+    "category": "Package",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "version": {
+        "Major": 0,
+        "Minor": 156,
+        "Patch": 0
+    },
+    "minimumAgentVersion": "2.105.7",
```

Comment:
recommend increasing this to 2.120.7 so that we always get telemetry

Reply:
will do